db/migrate/20210506161800_case_insensitive_user_emails.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CaseInsensitiveUserEmails < ActiveRecord::Migration[6.1]
+  def up
+    enable_extension("citext")
+
+    change_column :users, :email, :citext, null: false
+  end
+
+  def down
+    change_column :users, :email, :string, null: false, default: ""
+
+    disable_extension("citext")
```

Comment:
I feel this might cause trouble. See https://docs.cloud.service.gov.uk/deploying_services/postgresql/#:~:text=you%20cannot%20remove%20these%20mandatory%20extensions%20from%20a%20postgresql%20service%20instance

Reply:
Ah, interesting! Good catch :)

Reply:
OK, left the comment with the link. I still need to enable the extension so it works locally - `enable_extension` is nicely idempotent though so no risk for production db.